core/server/common/src/main/java/alluxio/master/journal/JournalMasterClientServiceHandler.java

```diff
@@ -87,4 +99,56 @@ public void getTransferLeaderMessage(GetTransferLeaderMessagePRequest request,
     RpcUtils.call(LOG, () -> mJournalMaster.getTransferLeaderMessage(request.getTransferId()),
             "GetTransferLeaderMessage", "request=%s", responseObserver, request);
   }
+
+  @Override
+  public void crashRatisServer(CrashRatisServerPRequest request, StreamObserver<CrashRatisServerPResponse> responseObserver) {
```

Comment:
i originally thought I can add a command to trigger a crash, then i realized standby masters don't take RPCs.. This will be removed.